index.html

```diff
@@ -371,56 +371,54 @@ <h4>Advanced</h4>
 				<div class="row text-center">
 					<p class="montserrat text-uppercase m-t-50">trusted by</p>
 				</div>
-
-				<div class="row no-gutter clients-bordered large-fix">
-
+				<div
+					class="row no-gutter clients-bordered large-fix text-center">
 					<div class="client col-md-2 col-sm-4 col-xs-6">
 						<a href="https://www.bitwala.io/" target="_blank"><img src="assets/images/clients/client-2.png" alt="Bitwala"></a>
 					</div>
-
 					<div class="client col-md-2 col-sm-4 col-xs-6">
 						<a href="https://purse.io" target="_blank"><img src="assets/images/clients/client-4.png" alt="Purse"></a>
 					</div>
-
 					<div class="client col-md-2 col-sm-4 col-xs-6">
 						<a href="https://www.ripio.com" target="_blank"><img src="assets/images/clients/client-5.png" alt="Ripio"></a>
 					</div>
-
 					<div class="client col-md-2 col-sm-4 col-xs-6">
 						<a href="https://btc.com/" target="_blank"><img src="assets/images/clients/client-6.png" alt="BTC.com"></a>
 					</div>
-
 					<div class="client col-md-2 col-sm-4 col-xs-6">
 						<a href="https://bixin.com/" target="_blank"><img src="assets/images/clients/client-7.png" alt="Bixin"></a>
 					</div>
-
-					<div class="col-md-2"></div>
-
 					<div class="client col-md-2 col-sm-4 col-xs-6">
 						<a href="https://bitpay.com/" target="_blank"><img src="assets/images/clients/client-1.png" alt="Bitpay"></a>
 					</div>
-
-					<div class="client col-md-2 col-sm-4 col-xs-6 col-md-offset-2">
+				</div>
+				<div
+					class="row no-gutter clients-bordered large-fix text-center text-center-md">
+					<div class="client col-md-2 col-sm-4 col-xs-6">
 						<a href="https://bitcore.io/" target="_blank"><img src="assets/images/clients/client-9.png" alt="Bitcore"></a>
 					</div>
-
 					<div class="client col-md-2 col-sm-4 col-xs-6">
 						<a href="https://copay.io" target="_blank"><img src="assets/images/clients/client-10.png" alt="Copay"></a>
 					</div>
-
-					<div class="col-sm-2 hidden-md hidden-lg hidden-xs"></div>
-
 					<div class="client col-md-2 col-sm-4 col-xs-6">
 						<a href="https://edgesecure.co/" target="_blank"><img src="assets/images/clients/client-11.png" alt="Edge Secure"></a>
 					</div>
-
 					<div class="client col-md-2 col-sm-4 col-xs-6">
 						<a href="https://chronobank.io/" target="_blank"><img src="assets/images/clients/client-12.png" alt="Chronobank.io"></a>
 					</div>
-				</div><!-- .row -->
-
+					<div class="client col-md-2 col-sm-4 col-xs-6">
+						<a href="https://redeeem.com/" target="_blank"><img src="assets/images/clients/client-13.png" alt="Redeeem"></a>
+					</div>
+				</div>
+				<!-- .row -->
 			</div>
 		</section>
+		<!-- Clients Css -->
+		<style>
+		.clients-bordered .client {padding:30px 10px;}
+		.clients-bordered .client:not(:last-child){border:none;}
+		@media only screen and (min-width: 992px) {.row.text-center-md > div{display:inline-block;float:none;}}
+		</style>
```

Comment:
Your layout looks great, thanks!
But I think including inline CSS like this is a bit of an anti-pattern. Could we move this to a css file in `assets/css`? and then link it the same way the bootstrap css is linked etc at the top of the page. I guess this is the first custom css on the front page so you'll need to start a new file, maybe call it `custom-frontpage.css` since we already have a `custom.css` for the guides.

If you don't have time for this, I can add the commit to this branch and merge sometime later this week.